### ISSUE TITLE
Default sample catalog/schema/tables not loading

### ISSUE THREAD

**Issue by avriiil** [unitycatalog/unitycatalog#145]
After pulling latest changes from `main` today I seem to have lost the toy data that was loading into my local UC 

Steps taken:

1. pull latest changes from `main`
2. spin up local UC server with `bin/start-uc-server`

Running `bin/uc catalog list` returns an empty list `[]`.

Running `bin/uc table list --catalog unity --schema default` returns:

```
Exception in thread "main" java.lang.RuntimeException: io.unitycatalog.client.ApiException: listTables call failed with: 404 - {"error_code":"NOT_FOUND","details":[{"reason":"NOT_FOUND","metadata":{},"@type":"google.rpc.ErrorInfo"}],"stack_trace":null,"message":"Catalog not found: unity"}
        at io.unitycatalog.cli.UnityCatalogCli.main(UnityCatalogCli.java:98)
Caused by: io.unitycatalog.client.ApiException: listTables call failed with: 404 - {"error_code":"NOT_FOUND","details":[{"reason":"NOT_FOUND","metadata":{},"@type":"google.rpc.ErrorInfo"}],"stack_trace":null,"message":"Catalog not found: unity"}
        at io.unitycatalog.client.api.TablesApi.getApiException(TablesApi.java:76)
        at io.unitycatalog.client.api.TablesApi.listTablesWithHttpInfo(TablesApi.java:342)
        at io.unitycatalog.client.api.TablesApi.listTables(TablesApi.java:317)
        at io.unitycatalog.cli.TableCli.listTables(TableCli.java:127)
        at io.unitycatalog.cli.TableCli.handle(TableCli.java:46)
        at io.unitycatalog.cli.UnityCatalogCli.main(UnityCatalogCli.java:82)
```

I've tried running `build/sbt package` after pulling latest changes, but this doesn't solve the problem.

Looks like I'm not the only one running into these issues, see [UC Slack](https://unitycatalog.slack.com/archives/C076YREKX8W/p1720057128650059)

**Comment by avriiil** [comment]
I can get the sample data working again by going back to `git checkout 22559918`.
It looks like the `etc/db/h2db.mv.db` file got deleted in #131, could this be related?

**Comment by vikrantpuppala** [comment]
Fixed in #102
